css/style.css

```
@@ -0,0 +1,353 @@
+/* Variables */
+:root {
+  --basic-red: #EE3643;
+  --basic-green: #63A63E;
+  --basic-grey: #3D546F;
+  --basic-greylight: #C5C5C5;
+  --basic-background: #F1F5F7;
+  --basic-white: #ffffff;
+  --special-ylellow: #FFBF47;
+  --success: #5FBB2D;
+  --error: #BA2732;
+  --basic-black: #000000;
+  --special: #663d15;
+  --focus-nav: #293449;
+  --address: #32425C;
+  --basic-blue: #3BBCE3;
+  --basic-purple: #DC91D8;
+  --green: #8ED78F;
+  --ylellow: #FFC047;
+  --special-gray: #F9F5F0;
+  --old-price: #A9A9A9;
+  --gold: #FFD180;
+  --focus-button: #CA2C37;
+}
+
+/* Global */
+body {
+  min-width: 1200px;
+  margin: 0;
+  padding: 0;
+  font-family: "Cuprum";
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 400;
+  color: var(--basic-black);
+  background-color: var(--basic-background);
+  background-position: top center;
+  background-repeat: no-repeat;
+}
+
+
+h2 {
+  Font family: Cuprum;
+  Font style: Regular;
+  Font size: 30px;
+  Line height: 30px;
+}
+
+.first-section {
+  background-color: var(--basic-background);
+}
+
+a {
+  text-decoration: none;
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.main-navigation {
+  background-color: var(--basic-grey);
+}
+
+.navigation {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  text-decoration: none;
+  list-style: none;
+}
+
+.color-nav {
+  color: var(--basic-white);
+}
+
+.main-navigation a,
+.catalog-list a,
+.list-orders a {
+  color: var(--basic-white);
+}
+
+/* header */
+.search {
+  Font family: Cuprum;
+  Font style: Regular;
+  Font size: 18px;
+  Line height: 24px;
+  background-color: var(--focus-nav);
+}
+
+.list-orders {
+  Font family: Cuprum;
+  Font style: Regular;
+  Font size: 18px;
+  Line height: 24px;
```

```suggestion
  Font family: Cuprum;
  Font style: Regular;
  Font size: 18px;
  Line height: 24px;
  background-color: var(--focus-nav);
}

.list-orders {
  font-family: Cuprum;
  Font style: Regular;
  Font size: 18px;
  Line height: 24px;
```
Посмотри вимательней на синтаксис. Таких правил нету